examples/platforms/eagle/Makefile.am

```diff
@@ -0,0 +1,62 @@
+#
```

Comment:
Could we add a README to `examples/platforms/eagle/` to describe how to compile and use the platform Eagle?